### PR TITLE
creates SlackUsers for all users with IDs filled in upon integration

### PR DESCRIPTION
**Pull request by ekim6767**
admin user still lacks the readable_name attribute, but does have its ID filled in

### CODE REVIEW

File: policykit/slackintegration/views.py [hunk 1]
```diff
@@ -79,42 +81,54 @@ def oauth(request):
...
+                #get the list of users, create SlackUser object for each user
+                data2 = parse.urlencode({
+                    'token':community.access_token
+                }).encode()
+                                       
+                req2 = urllib.request.Request('https://slack.com/api/users.list', data=data2)
+                resp2 = urllib.request.urlopen(req2)
+                res2 = json.loads(resp2.read().decode('utf-8'))
+                
+                #https://api.slack.com/methods/users.list
```
Comment (amyxzhang) [inline policykit/slackintegration/views.py:99]: just checking, @ekim6767 - is users.list an admin-only API request? if so, then we don't even need the below call to users.info that checks if it returns an admin error, can just use the result from this call?

Comment (ekim6767) [reply policykit/slackintegration/views.py:99]: I believe that it works with the user, bot and admin tokens, and i'm using the bot token in the code, but I think there is a way to integrate the user.info call with it

Comment (amyxzhang) [reply policykit/slackintegration/views.py:99]: ah well if there's not a way to do check/determination for adminship using users.list then let's forget about itt.

File: policykit/slackintegration/views.py [hunk 2]
```diff
@@ -137,6 +151,22 @@ def is_policykit_action(integration, test_a, test_b, api_name):
 
     return False
 
+#creates slackUser object along with the readable name and user_id
```
Comment (amyxzhang) [inline policykit/slackintegration/views.py:154]: you should remove this

File: policykit/slackintegration/views.py [hunk 3]
```diff
@@ -178,19 +210,23 @@ def action(request):
                 new_api_action.text = event['text']
                 new_api_action.channel = event['channel']
                 new_api_action.time_stamp = event['ts']
-                u,_ = SlackUser.objects.get_or_create(username=event['user'],
-                                                                           community=community)
+                
+                #u = create_slack_user(community, event['user'])
```
Comment (amyxzhang) [inline policykit/slackintegration/views.py:214]: same for the rest of the commented out code below

File: policykit/slackintegration/views.py (hunk 2 above)
Comment (ekim6767) [reply policykit/slackintegration/views.py:154]: this was the start of my solution to #6 which I don't think I got working yet, but I will send a new request without it

**Pull request merged**